driver/docker-compose.yml

```diff
@@ -11,17 +11,16 @@ services:
   truffle:
     command: /app/run.sh
     build:
-      context: .
-      dockerfile: docker/truffle/Dockerfile
+      context: docker/truffle
```

Comment:
nit: Personally I would prefer that the context just be the root of the project. My main argument is that then all paths are relative to the root of the project and no path math is needed.

Reply:
Yeah I'm unsure about this. I don't mind either way. Since the truffle docker doesn't need any file from anywhere else I did this which has the advantage of  staying correct even when /driver/docker/truffle moves. As I said I know what's better so let's see what the second reviewer thinks.

Reply:
> which has the advantage of staying correct even when /driver/docker/truffle moves

Thats a good point

> Yeah I'm unsure about this

> As I said I know what's better

:rofl:

Reply:
Added `don't` :)

Reply:
Hehe, I figured that's what you meant. I just thought it was funny :stuck_out_tongue_closed_eyes: